### ISSUE TITLE
missing `libgfortran.so.5`

### ISSUE THREAD

**Issue by apozharski** [casadi/casadi#3102]
Hello,

A clean install of casadi3.6.0 via `pip install casadi` (tested on python3.7 through python3.10) fails when trying to use `nlpsol` with the `ipopt` plugin. The error is the following (in 3.10  but the others are similar):

```
CasADi - 2023-04-13 17:52:38 WARNING(".../casadi/core/casadi_os.cpp:166: Assertion "handle!=nullptr" failed:
PluginInterface::load_plugin: Cannot load shared library 'libcasadi_nlpsol_ipopt.so': 
   (
    Searched directories: 1. casadipath from GlobalOptions
                          2. CASADIPATH env var
                          3. PATH env var (Windows)
                          4. LD_LIBRARY_PATH env var (Linux)
                          5. DYLD_LIBRARY_PATH env var (osx)
    A library may be 'not found' even if the file exists:
          * library is not compatible (different compiler/bitness)
          * the dependencies are not found
   )
  Tried '/home/anton/syscop/nosnoc_py/env/lib/python3.10/site-packages/casadi' :
    Error code: libgfortran.so.5: cannot open shared object file: No such file or directory
  Tried '' :
    Error code: libgfortran.so.5: cannot open shared object file: No such file or directory
  Tried '.' :
    Error code: ./libcasadi_nlpsol_ipopt.so: cannot open shared object file: No such file or directory") [.../casadi/core/plugin_interface.hpp:142]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/anton/syscop/nosnoc_py/env/lib/python3.10/site-packages/casadi/casadi.py", line 28924, in nlpsol
    return _casadi.nlpsol(*args)
RuntimeError: .../casadi/core/plugin_interface.hpp:292: Plugin 'ipopt' is not found.
```

It seems like the version of the `libgfortran` binary that is packaged with the install is `libgfortran.so.3` and not the expected `libgfortran.so.5` and might be related to the closed #3032.

**Comment by jgillis** [comment]
Thanks for reporting this fundamental issue. I thought I fixed this but I can confirm your findings..

**Comment by KaramAlmaghout** [comment]
I had the same issue with Ubuntu 18.04, python3.10, casadi3.6

the problem is solved by installing libgfortran.so.3

`sudo apt-get install libgfortran3`

**Comment by jgillis** [comment]
Fixed in 3.6.1
